site/content/en/docs/tasks/manage/setup_object_retention_policy.md

```diff
@@ -0,0 +1,83 @@
+---
+title: "Setup object retention"
```

Comment:
```suggestion
title: "Setup garbage-collection of workloads"
```

"Setup object retention" is not very descriptive for user docs.

Reply:
Sure, have updated the title